python/Ganga/Utility/execute.py

```diff
@@ -194,7 +195,7 @@ def execute(command,
         python_setup (str): A python command to be executed beore the main command is
         eval_includes (str): An string used to construct an environment which, if passed, is used to eval the stdout into a python object
         update_env (bool): Should we update the env being passed to what the env was after the command finished running
-        return_code (int): This is the returned code from the command which is executed
+        return_code (list): This has element 0 modified to be the returned code from the command which is executed
```

Comment:
This is not a nice API to use for this, it is far too likely to be used incorrectly which will cause silent failures. This makes much more sense to be done as a return value of the function (perhaps as a `{'rc':..., 'stdout':..., 'stderr':...}` dictionary or maybe better a named tuple). Using function arguments as a return mechanism is not recommended.